### DISCUSSION TITLE
Websocket Connection To Walletconnect Failed

### DISCUSSION THREAD

**Discussion by petermazzocco** [wevm/wagmi#2831] [General]
Just recently I started getting a flood of errors for Websocket not connecting to WalletConnect:

`Websocket.js?3450:82 WebSocket connection to 'wss://4.bridge.walletconnect.org/?env=browser&host=localhost%3A3000&protocol=wc&version=1' failed: `

My WC ID is valid so I'm just assuming something is off on the wagmi end and wanted to bring attention to it.

If not, does anyone know the solution?

**Comment by wushaobo** [comment]
I am experiencing the same issue. The websocket is trying to connect the following but failed
- `d.bridge.walletconnect.org`
- `2.bridge.walletconnect.org`
- `r.bridge.walletconnect.org`
- ......

The DNS cannot be resolved.

```
➜  nslookup r.bridge.walletconnect.org 8.8.8.8
Server:		8.8.8.8
Address:	8.8.8.8#53

** server can't find r.bridge.walletconnect.org: NXDOMAIN

➜  nslookup bridge.walletconnect.org 8.8.8.8
Server:		8.8.8.8
Address:	8.8.8.8#53

Non-authoritative answer:
*** Can't find bridge.walletconnect.org: No answer

➜  nslookup walletconnect.org 8.8.8.8
Server:		8.8.8.8
Address:	8.8.8.8#53

Non-authoritative answer:
Name:	walletconnect.org
Address: 104.18.21.250
Name:	walletconnect.org
Address: 104.18.20.250
```

**Reply by abhimarshal1** [reply]
This is because, the bridge is for wallet connect v1 which has been shutdown. Upgrade to wallet connect v2.

**Comment by mesquitaoliveira** [comment]
> Just recently I started getting a flood of errors for Websocket not connecting to WalletConnect:
> 
> `Websocket.js?3450:82 WebSocket connection to 'wss://4.bridge.walletconnect.org/?env=browser&host=localhost%3A3000&protocol=wc&version=1' failed: `
> 
> My WC ID is valid so I'm just assuming something is off on the wagmi end and wanted to bring attention to it.
> 
> If not, does anyone know the solution?

I am using `ConnectKit` and I received the same error! I solved it in the following way:
**info: Vite + React js**
`package.json`:

```json
...
"connectkit": "^1.6.0",
 "viem": "1.x",
 "wagmi": "1.x",
 ...
```

`main.jsx`:
```jsx
...

const config = createConfig(
  getDefaultConfig({
    // Required API Keys
    alchemyId: process.env.ALCHEMY_ID, // or infuraId
    walletConnectProjectId = import.meta.env.VITE_WALLETCONNECT_PROJECT_ID;,
    ...
  }),
);
```
